sdk/test-utils/perf/GettingStarted.md

```diff
@@ -65,7 +63,7 @@ To add perf tests for the `sdk/<service>/<service-sdk>` package, follow the step
     ```
 
 5.  Run `rush update` and commit the changes to the `pnpm-lock` file.
-6.  Copy the `tsconfig.json`, `sample.env`(and `.env`) files that are present at the `sdk/<service>/<service-sdk>` to `sdk/<service>/perf-tests/<service-sdk>`.
+6.  Copy the `tsconfig.json`, `sample.env`(and `.env`) files that are present at the `sdk/<service>/<service-sdk>` to `sdk/<service>/<service-sdk>-perf-tests`.
 
     TSCONFIG
```

Comment:
Should fix the TSConfig section to copy over the tsconfig.json/tsconfig.src.json which point to the base instead of setting its own parameters.

Reply:
updated